modules/editor-events/module.php

```diff
@@ -4,23 +4,32 @@
 
 use Elementor\Core\Base\Module as BaseModule;
 use Elementor\Core\Common\Modules\Connect\Apps\Base_App;
-use Elementor\Tracker;
+use Elementor\Core\Experiments\Manager as Experiments_Manager;
 use Elementor\Utils;
+use Elementor\Plugin;
+use Elementor\Tracker;
 
 if ( ! defined( 'ABSPATH' ) ) {
 	exit; // Exit if accessed directly.
 }
 
 class Module extends BaseModule {
 
-	const ELEMENTOR_EDITOR_EVENTS_DATA_SYSTEM_URL = 'https://my.elementor.com/api/v1/editor-events';
+	const EXPERIMENT_NAME = 'editor_events';
+
+	public function __construct() {
+		parent::__construct();
+
+		$this->register_experiment();
+	}
 
 	public function get_name() {
 		return 'editor-events';
 	}
 
 	public static function get_editor_events_config() {
-		$can_send_events = false;
+		$can_send_events = Tracker::is_allow_track() &&
+			Plugin::$instance->experiments->is_feature_active( self::EXPERIMENT_NAME ) && defined( 'ELEMENTOR_EDITOR_EVENTS_MIXPANEL_TOKEN' );
```

Comment:
Check `defined( 'ELEMENTOR_EDITOR_EVENTS_MIXPANEL_TOKEN' )` first, then it will avoid 2 database calls for normal users.